docker-compose.yml

```diff
@@ -69,7 +69,7 @@ services:
       govwifi-fake-s3:
         condition: service_healthy
     environment:
-      DEFAULT_REGION: eu-west-1
+      AWS_DEFAULT_REGION: eu-west-1
```

Comment:
Just want to check this is the right region (I know it didn't change, but just want to double check), eu-west-1 is ireland, Dublin, sure we want this and not the default eu-west-2 London?

Reply:
it's a Fake S3 instance